frontend/packages/console-shared/src/components/dashboard/utilization-card/utilization-card.scss

```diff
@@ -36,14 +36,30 @@
 }
 
 .co-utilization-card__item .co-utilization-card__item__chart.co-utilization-card__item__chart--times {
-  padding: var(--pf-global--spacer--md) var(--pf-global--spacer--md) 0 var(--pf-global--spacer--md);
+  padding: var(--pf-global--spacer--sm) var(--pf-global--spacer--lg) var(--pf-global--spacer--sm) var(--pf-global--spacer--md);
+  svg {
+    overflow: visible;
+  }
 }
 
 .co-utilization-card__item__chart .graph-wrapper {
   margin: 0;
   padding: 0;
 }
 
+.co-utilization-card__item-section {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.co-utilization-card__item-description {
```

Comment:
Nit:  alphabetize rules by selector (this rule should come before the addition on line 50.

Reply:
fixed